packages/components/src/components/textarea/textarea.test.ts

```diff
@@ -167,10 +167,24 @@ describe('<sd-textarea>', () => {
       await sendKeys({ press: 'a' });
       await sendKeys({ press: 'Backspace' });
       await el.updateComplete;
+      el.blur();
+      await el.updateComplete;
 
       expect(el.hasAttribute('data-user-invalid')).to.be.true;
       expect(el.hasAttribute('data-user-valid')).to.be.false;
     });
+
+    // it('should receive validation attributes ("states") even when novalidate is used on the parent form', async () => {
+    //   const el = await fixture<HTMLFormElement>(html` <form novalidate><sd-textarea required></sd-textarea></form> `);
+    //   const textarea = el.querySelector<SdTextarea>('sd-textarea')!;
+    //
+    //   expect(textarea.hasAttribute('data-required')).to.be.true;
+    //   expect(textarea.hasAttribute('data-optional')).to.be.false;
+    //   expect(textarea.hasAttribute('data-invalid')).to.be.true;
+    //   expect(textarea.hasAttribute('data-valid')).to.be.false;
+    //   expect(textarea.hasAttribute('data-user-invalid')).to.be.false;
+    //   expect(textarea.hasAttribute('data-user-valid')).to.be.false;
+    // });
```

Comment:
Why did you uncomment this test?

Reply:
This test ist still not working, but I'm on it

Reply:
@abudd1094 will take care of this as part of the validation stuff he is fixing.

Reply:
pls add a hint to the code as mentioned in the contribution guide => https://github.com/solid-design-system/solid/pull/618/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R152